BeforeBuild.cmd

```diff
@@ -0,0 +1,35 @@
+::Configuration for AppVeyor build platform
+
+@ECHO off
+SETLOCAL
+
+if NOT exist "GeneratedKey.snk" (
+sn -k GeneratedKey.snk
+)
+
+if "%ERRORLEVEL%" NEQ "0" (
+echo COMMAND sn -k FAILED
+goto ExitFailed
+)
+
+if NOT exist "GeneratedKey.snk" (
+echo GeneratedKey.snk NOT FOUND
+goto ExitFailed
+)
+
+@REM Restore nuget packages
+%~dp0.nuget\NuGet.exe restore src\Everything.sln -ConfigFile .nuget\NuGet.Config
+
+if "%ERRORLEVEL%" NEQ "0" (
+echo NUGET RESTORE FAILED
+goto ExitFailed
+)
+
+goto Exit
+
+:ExitFailed
+@echo.
+@echo SCRIPT FAILED
```

Comment:
You might change this to "Script %~n0 failed".  %~n0 resolves to the file name of the script, so it should show up in the log as "Script BeforeBuild.cmd failed".